### ISSUE TITLE
Texture errors in TMPE.log

### ISSUE THREAD

**Issue by aubergine10** [CitiesSkylinesMods/TMPE#704]
I'm guessing these are result of tryng to find localised images for TTL UI and won't acutally cause any problems?

EDIT: Actually, that usually creates just a warning - see: #468

```diff
Error 1,843.4759762:   at TrafficManager.UI.Textures.TextureResources.ReadToEnd (System.IO.Stream stream) [0x00000] in <filename unknown>:0 
  at TrafficManager.UI.Textures.TextureResources.LoadDllResource (System.String resourceName, Int32 width, Int32 height) [0x00000] in <filename unknown>:0 
   at CSUtil.Commons.Log.LogToFile(System.String log, LogLevel level)
   at CSUtil.Commons.Log.Error(System.String s)
   at TrafficManager.UI.Textures.TextureResources.LoadDllResource(System.String resourceName, Int32 width, Int32 height)
+   at TrafficManager.UI.Textures.TrafficLightTextures..cctor()
+   at TrafficManager.UI.SubTools.TimedTrafficLightsTool.ShowGUIOverlay(ToolMode toolMode, Boolean viewOnly)
+   at TrafficManager.UI.SubTools.TimedTrafficLightsTool.ShowGUIOverlay(ToolMode toolMode, Boolean viewOnly)
   at TrafficManager.UI.TrafficManagerTool.OnToolGUI(UnityEngine.Event e)
   at ToolBase.OnGUI()
```

[TMPE(2).log](https://github.com/krzychu124/Cities-Skylines-Traffic-Manager-President-Edition/files/4209563/TMPE.2.log)
[output_log(12).txt](https://github.com/krzychu124/Cities-Skylines-Traffic-Manager-President-Edition/files/4209564/output_log.12.txt)

**Comment by aubergine10** [comment]
Potential mod conflicts:

* 554232266    SeniorCitizenCenterMod -- confirmed to cause some errors, but not the one specifically mentioned in OP
* 408905948    Pause on load -- can cause game to get stuck on pause (unable to unpause)
* 694123443    American Traffic Lights -- might cause TTL texture issue somehow?
* 1922400472   Enhanced District Services 1.0.18 -- this mod also features in #714
* 810858473    Traffic Report Tool 2.0 -- somewhat obsolete, might cause issues?
* 1651036644   Traffic Congestion Report -- potential conflict with TMPE which also tracks that data?
* 1934023593   RM Crossings V2.5 -- ?
* 638786709    Local-Only Passenger Harbours and Airports -- ?
* 747836519    AssetIconsV2 -- known game-breaking mod

**Comment by aubergine10** [comment]
This should hopefully be fixed in the 11.1.1 release.
